### PR TITLE
Updated Socket

### PR DESCRIPTION
**Pull request by olapinska**

### CODE REVIEW

**Review by ksyhoo — APPROVED** [review]

File: test/__mocks__/fb.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
+import fb from 'fb'
```
Comment (mkucharz) [inline test/__mocks__/fb.js]: Remove this file is a FB mock, it is in boilerplate as an example.

File: socket.yml [hunk 2]
```diff
@@ -2,27 +2,27 @@ name: markdown-converter
...
+ - markdown
+ - html
+ - converter
 endpoints:
-  convert:
-    description: Convert Markdown to HTML
-    file: scripts/convert.js
+ convert:
+   description: Convert Markdown to HTML
+   input:
```
Comment (mkucharz) [inline socket.yml]: We have `inputs` and `outputs`.

File: test/__mocks__/axios.js [hunk 3]
```diff
@@ -0,0 +1,2 @@
+import axios from 'axios'
```
Comment (mkucharz) [inline test/__mocks__/axios.js]: We are not using axios, so we don't need mock also.

**Review by mkucharz — CHANGES_REQUESTED** [review]

File: package.json [hunk 4]
```diff
@@ -1,6 +1,90 @@
...
+  "repository": "https://github.com/eyedea-io/syncano-socket-markdown-converter.git",
+  "public": true,
+  "author": "Eyedea <hello@eyedea.io> https://eyedea.io",
+  "license": "MIT",
+  "keywords": [
+    "syncano",
+    "socket"
+  ],
+  "peerDependencies": {
+    "@syncano/cli": "0.13.0-alpha.f6ea95d4"
```
Comment (Idered) [inline package.json]: This should be:

```
"@syncano/cli": "^0.13.0-alpha.f6ea95d4"
```

Which means version `0.13.0-alpha.f6ea95d4` or higher `^`

File: socket.yml [hunk 5]
```diff
@@ -2,27 +2,27 @@ name: markdown-converter
...
+    success:
+      exit_code: 200
+      description: Success
+      example: <p>Hello <em>World</em>!</p>
+    fail:
+      exit_code: 400
+      description: Failed
+      example: |
+        {
+          “message”: “Conversion failed!”
```
Comment (Idered) [inline socket.yml]: Invalid quotation marks `“` should be `"`

**Review by Idered — CHANGES_REQUESTED** [review]

File: socket.yml [hunk 6]
```diff
@@ -2,27 +2,27 @@ name: markdown-converter
 description: Simple Markdown do HTML converter
```
Comment (mkucharz) [inline socket.yml]: Obsolete  description and version. Move keywords to `package.json`.

File: src/convert.ts [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+    {response}: S.Core,
+    {args}: S.Context<Args>
+  ) {
+    try {
+      response(markdown.toHTML(args.content), 200, 'text/html')
+     } catch(err) {
+      response.json({message: 'Conversion failed!'}, 400)
+     }
+  }
+
```
Comment (mkucharz) [inline src/convert.ts]: Remove empty line.

**Review by mkucharz — CHANGES_REQUESTED** [review]

**Review by jonny22094 — APPROVED** [review]

**Pull request merged**